src/chrome/chromeDebugAdapter.ts

```diff
@@ -584,7 +588,7 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
 
         this._expectingStopReason = undefined;
 
-        smartStepP.then(should => {
+        await smartStepP.then(should => {
```

Comment:
This change is intentional. We should take regard of the time spent on this operation when we measure `Debugger.paused` event.